### ISSUE TITLE
bug compiling SDK in Eclipse

### ISSUE THREAD

**Issue by gavinking** [ceylon/ceylon-ide-eclipse#1390]
When I compile the `ceylon-sdk` project in Eclipse I get:

```
Cannot find symbol
  symbol:   class AssertionError$header$
  location: package ceylon.language
```

**Comment by davidfestal** [comment]
@gavinking : I don't get this error exactly. Could tell me on which file you encounter it ?

**Comment by gavinking** [comment]
I get it in `source/ceylon/test/assertions.ceylon`.

**Comment by davidfestal** [comment]
I see it now.
And it seems it's related the natives, and is probably a site-effect of the temporary fix (ceylon/ceylon-ide-eclipse@3a7c898) for issue #1380 :-(

**Comment by gavinking** [comment]
Yeah, it's very new.

**Comment by davidfestal** [comment]
Confirmed. So to fix this I need removing the hack that was fixing #1380.
I'll have to fix #1380 with the right way, which seem also related to a remaining bug in native support, relative to the type of native declarations.
